src/alembic/versions/c8eb0d9596a7_migrate_ligt_in_ggwpgebied.py

```diff
@@ -0,0 +1,144 @@
+        sa.Column('src_id', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('src_volgnummer', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('bronwaarde', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('derivation', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('dst_source', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('dst_id', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('dst_volgnummer', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('begin_geldigheid', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('eind_geldigheid', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_last_src_event', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('_last_dst_event', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('_source', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_application', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_source_id', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_last_event', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('_hash', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_version', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_date_created', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_date_confirmed', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_date_modified', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_date_deleted', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_expiration_date', sa.DateTime(), autoincrement=False, nullable=True),
+        sa.Column('_gobid', sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column('_id', sa.String(), autoincrement=False, nullable=True),
+        sa.Column('_tid', sa.String(), autoincrement=False, nullable=True),
+        sa.ForeignKeyConstraint(['dst_id', 'dst_volgnummer'], ['gebieden_ggwgebieden._id', 'gebieden_ggwgebieden.volgnummer'], name='rel_gbd_wijk_gbd_ggw_ligt_in_ggwgebied_dfk'),
+        sa.ForeignKeyConstraint(['src_id', 'src_volgnummer'], ['gebieden_wijken._id', 'gebieden_wijken.volgnummer'], name='rel_gbd_wijk_gbd_ggw_ligt_in_ggwgebied_sfk'),
+        sa.PrimaryKeyConstraint('_gobid'),
+        sa.UniqueConstraint('_source_id', name='rel_gbd_wijk_gbd_ggw_ligt_in_ggwgebied_uniq'),
+        sa.UniqueConstraint('_tid', name='rel_gbd_wijk_gbd_ggw_ligt_in_ggwgebied__tid_key')
+    )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.rename_table("rel_gbd_brt_gbd_ggp__ligt_in_ggpgebied", "rel_gbd_brt_gbd_ggp_ligt_in_ggpgebied")
```

Comment:
This will not work. The rename will collide with existing tables. 
There is already a rel_gbd_brt_gbd_ggp_ligt_in_ggpgebied, created in the forward migration.

To get this to work, first rel_gbd_brt_gbd_ggp_ligt_in_ggpgebied has to be deleted.

Reply:
fixed